code/modules/cards/singlecard.dm

```diff
@@ -48,6 +48,9 @@
 		if(parent_deck.holodeck)
 			flags_1 |= HOLOGRAM_1
 			parent_deck.holodeck.spawned += src
+	if(mapload)
+		//maploaded card needs to be faceup anyways, and doing this will give it its name and appearance properly
+		Flip(TRUE)
```

Comment:
The only purpose of this is to make the map work, not a feature or anything. There aren't any naturally spawning singlecards